apps/state/config/config.exs

```diff
@@ -12,6 +12,8 @@ config :state, :route_pattern,
     "Green-D-1-1" => true,
     "Green-D-3-1" => true,
     # don't ignore Rockport Branch shuttles
+    "Shuttle-BeverlyRockport-0-0" => false,
+    "Shuttle-BeverlyRockport-0-1" => false,
```

Comment:
This is so that the Beverly–Rockport shuttles get displayed on the dotcom timetable for `CR-Newburyport`.